sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -547,264 +547,6 @@ class RocksDBSuite extends AlsoTestWithChangelogCheckpointingEnabled with Shared
     }
   }
 
-  testWithColumnFamilies(s"RocksDB: column family creation with invalid names",
```

Comment:
These tests in RocksDBSuite are removed and moved into `RocksDBStateStoreSuite` because the verification functions are moved from `RocksDB` to `RocksDBstateStoreProvider`.